build/build.cake

```diff
@@ -22,11 +22,11 @@ var sonarToken = HasArgument("sonar-token")
 var srcPath        = "./../";
 var srcDir         = Directory(srcPath);
 var buildDir       = srcDir + Directory("bin/") + Directory(configuration);
-var webBuildDir    = srcDir + Directory("VolleyManagement.UI/bin");
+var webBuildDir    = srcDir + Directory("src/VolleyManagement.Backend/VolleyManagement.UI/bin");
```

Comment:
why we added "src/" only here but not for buildDir?

Reply:
Because path to build folder (folders with Debug/Release/UT dlls) is "rootFolder\bin" and for web build folder path is "rootFolder\src\VolleyManagement.Backend\VolleyManagement.UI\bin"

Reply:
then we need to rename "srcPath" to "rootPath" to better reveal intent. Adding "src" to srcPath variable seems strange